9.0-jre8/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG BASE_IMAGE=appsvc/java:8-jre8_0000000000
+FROM $BASE_IMAGE
+MAINTAINER Azure App Services Container Images <appsvc-images@microsoft.com>
+
+ENV AI_VERSION 2.1.0
```

Comment:
>2.1.0 [](start = 15, length = 5)

Please make sure we update AI to the latest version

Reply:
Done!